src/styleUtils.js

```diff
@@ -108,37 +106,14 @@ define(
 						});
 
 						saveButton.children().remove();
-
-						saveButton.downloadify({
-							swf : '../external/downloadify/downloadify.swf',
-							downloadImage : '../external/downloadify/download.png',
-							width : 100,
-							height : 30,
-							filename : filename,
-							data : cslCode,
-							transparent : true,
-							onComplete: function () {
-								alert('Your CSL Style Has Been Saved');
-								dialog.dialog('destroy');
-							},
-							onCancel: function () { },
-							onError: function () { alert('Error saving file.'); }
+						
+						dialog.find('#download').html('<button id="download-button">Save To Disk</button>');
```

Comment:
> You can use saveButton instead of dialog.find('#download').

Ah, right. Good catch.

> BTW is "Save to Disk" nowadays still the best choice for the label here?

It seems like that was the original label (see also https://github.com/citation-style-language/csl-editor-demo-site/pull/13), but I agree that "Download" or "Download Style" would be better.